### PR TITLE
(feat) Implement MySQL database structure with sequelize

### PR DESCRIPTION
**Pull request by rjatkinson2**

### CODE REVIEW

File: server/db/models/Gig.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+module.exports = function (sequelize, DataTypes) {
+  var Gig = sequelize.define('Gig', {
+    title: DataTypes.STRING,
+    type: DataTypes.ENUM('wedding', 'birthday', 'business'),
+    date: DataTypes.DATE,
+    start_time: DataTypes.TIME,
+    end_time: DataTypes.TIME,
+    complexity: DataTypes.INTEGER,
```
Comment (joshuaNewman10) [inline server/db/models/Gig.js:8]: Is complexity something we are going to be calculating or just some heuristic inputted by an admin?

File: server/db/models/Gig.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+module.exports = function (sequelize, DataTypes) {
+  var Gig = sequelize.define('Gig', {
+    title: DataTypes.STRING,
+    type: DataTypes.ENUM('wedding', 'birthday', 'business'),
```
Comment (joshuaNewman10) [inline server/db/models/Gig.js:4]: We should chat about Gig.type once we get some of the real data. I can see us running into problems trying to scale this and also running into problems when our app gets used by different catering companies

File: server/db/models/User.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+module.exports = function (sequelize, DataTypes) {
```
Comment (joshuaNewman10) [inline server/db/models/User.js:1]: Are all users of the site going to be admins?

**Comment by joshuaNewman10** [comment]
Ryan this looks great overall, just a few points I raised that I would like to hear your thoughts on. When you have commented let me know and I will merge in.

File: server/db/models/Gig.js (hunk 2 above)
Comment (rjatkinson2) [inline server/db/models/Gig.js:4]: Yea good call. I can normalize this into a separate table and submit a follow-up. That'll end up being the correct long term implementation.

File: server/db/models/Gig.js (hunk 1 above)
Comment (rjatkinson2) [inline server/db/models/Gig.js:8]: Yea it sounds like the sales team makes that subjective call based on the gig requirements when they put it into the system.

File: server/db/models/User.js (hunk 3 above)
Comment (rjatkinson2) [inline server/db/models/User.js:1]: No so the Role.hasMany(User) relationship adds a role_id to the User table which will specify admin and staff booleans for each user. When we do contractor uploads or they login through the contractor side of the application, it should automatically be set to admin=false. Admin = true should only be associated with the users creating an organization for the first time or set by the original admin manually.

**Comment by rjatkinson2** [comment]
@joshuaNewman10 comments are up. Let me know what you think. Thanks for the review!

**Comment by joshuaNewman10** [comment]
Awesome, all looks good, merging this in!

**Pull request merged**